pstress/pstress_log_to_sql_converter.sh

```diff
@@ -0,0 +1,56 @@
+###############################################################################################
+#                                                                                             #
+# Author  :  Mohit Joshi                                                                      #
+# Script  :  pstress_log_to_sql_converter.sh                                                  #
+# Created :  05-May-2022                                                                      #
+# Purpose :  The script is used to process pstress thread logs to filter out                  #
+#            SQLs from it                                                                     #
+#                                                                                             #
+# Usage   :                                                                                   #
+# ./pstress_log_to_sql_converter.sh default.node.tld_step_1_thread-0.sql                      #
+#                                                                                             #
+# Update the BASEDIR and SOCKET information of the running server before executing the script #
+#                                                                                             #
+###############################################################################################
+
+logFileName=$1
+
+SCRIPT=$(readlink -f $0)
+SCRIPT_PATH=`dirname $SCRIPT`
+outputFileName=$SCRIPT_PATH/reduced.sql
+
+echo "Reading the pstress logfile: $logFileName"
```

Comment:
Validate if file is empty or not.

Reply:
If the input file is empty, then the output file will also be empty.

Reply:
ensure it error out if file doesn't error.

Reply:
Fixed